### PR TITLE
create a hybrid cluster and run the e2e tests on it

### PR DESCRIPTION
**Pull request by JacobTanenbaum**

### CODE REVIEW

**Comment by JacobTanenbaum** [comment]
@stevekuznetsov PTAL

File: ci-operator/step-registry/ovn/install-hybrid/install/ovn-install-hybrid-install-commands.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+mkdir -p ~/.ssh
+cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
+
+
+# testing putting the create manifests and stuff here.
+cp "${SHARED_DIR}/install-config.yaml" "${dir}/"
+
+TF_LOG=debug openshift-install --dir="${dir}" create manifests
+
+cat >> "${dir}/manifests/cluster-network-03-config.yml" << EOF
```
Comment (stevekuznetsov) [inline ci-operator/step-registry/ovn/install-hybrid/install/ovn-install-hybrid-install-commands.sh:44]: I would much rather see you break apart the current install step into a manifest-create and a run-install or something. Whenver we find ourselves copy-pasting in this system we _need_ to be refactoring to make this reusable

Comment (JacobTanenbaum) [reply ci-operator/step-registry/ovn/install-hybrid/install/ovn-install-hybrid-install-commands.sh]: updated with the code split into steps

File: ci-operator/step-registry/ovn/conf/manifests/ovn-conf-manifests-commands.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```
Comment (stevekuznetsov) [inline ci-operator/step-registry/ovn/conf/manifests/ovn-conf-manifests-commands.sh:1]: I meant -- *everyone* running the installer should ideally be using the same steps to do so, or we have drift. So this should underpin the IPI install flow, too.

File: ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml [hunk 3]
```diff
@@ -7,4 +7,4 @@ ref:
       cpu: 1000m
       memory: 2Gi
   documentation: |-
-    The IPI install step runs the OpenShift Installer in order to bring up an OpenShift cluster, using the provided cluster profile to choose a target IaaS platform.
+    The IPI install step runs the OpenShift Installer in order to bring up an OpenShift cluster, using the provided cluster profile to choose a target IaaS platform. Anything that needs to be configured using manifests should generate them before this step and put them in the SHARED_DIR with the filename manifest_(manifest_name).yml so that this step can pull in the manifest in.
```
Comment (stevekuznetsov) [inline ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml:10]: @AlexNPavel can we embed HTML here or something? I would love to see a fully-fledged example.

Comment (abhinavdahiya) [reply ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml:10]: we could allow markdown and use https://godoc.org/github.com/gomarkdown/markdown/html to convert to html for `steps.svc.`

**Pull request merged**